packages/stellar-axelar-std-derive/src/storage.rs

```diff
@@ -198,42 +213,50 @@ fn public_storage_fns(
     };
 
     match &value {
-        Value::Status => value_status_fns(variant, &param_list, &storage_method, key),
+        Value::Status => value_status_fns(
```

Comment:
move this to an `impl Value` block. `storage_type` can be passed instead of storage_method and ttl separately